docs/monitoring/operator_alerts.md

```diff
@@ -7,9 +7,6 @@
 |KubePersistentVolumeFullInFourDays|warning|seed|`Based on recent sampling, the PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} is expected to fill up within four days. Currently {{ printf "%0.2f" $value }}% is available.`|
 |KubePodPendingControlPlane|warning|seed|`Pod {{ $labels.pod }} is stuck in "Pending" state for more than 30 minutes.`|
 |KubePodNotReadyControlPlane|warning||`Pod {{ $labels.pod }} is not ready for more than 30 minutes.`|
-|KubeStateMetricsShootDown|info|seed|`There are no running kube-state-metric pods for the shoot cluster. No kubernetes resource metrics can be scraped.`|
-|KubeStateMetricsSeedDown|critical|seed|`There are no running kube-state-metric pods for the seed cluster. No kubernetes resource metrics can be scraped.`|
-|NoWorkerNodes|blocker||`There are no worker nodes in the cluster or all of the worker nodes in the cluster are not schedulable.`|
```

Comment:
I think @wyb1 and @istvanballok are aware (this is not new and already happened with previous refactorings). Personally, I doubt whether anybody is relying on this doc since there were no questions or complaints about it.

Reply:
I'm also not sure what the future of that doc should be since it is not complete due to the monitoring configuration being spread out across components. The doc and script will need to be reworked anyway once GEP19 is implemented.